packages/inference/src/InferenceClient.ts

```diff
@@ -1,8 +1,9 @@
-import * as tasks from "./tasks";
-import type { Options } from "./types";
-import { omit } from "./utils/omit";
-import { typedEntries } from "./utils/typedEntries";
+import * as tasks from "./tasks/index.js";
+import type { Options } from "./types.js";
+import { omit } from "./utils/omit.js";
+import { typedEntries } from "./utils/typedEntries.js";
 
+/import\s+([^\s]+)\s+from\s+(['"])(\.{1,2}\/[^'"]+?)(?<!\.js)\2/g;
```

Comment:
Why is this needed?

Reply:
nvm it's the regex I used to replace the imports

Reply:
```suggestion
```